media_driver/agnostic/gen11/codec/hal/codechal_kernel_header_g11.h

```diff
@@ -129,15 +162,16 @@ static MOS_STATUS GetCommonKernelHeaderAndSizeG11(
     case VDENC_ME:
         currKrnHeader = &kernelHeaderTable->hmeVdenc;
         break;
+    case ENC_SCALING_CONVERSION:
+        currKrnHeader = &kernelHeaderTable->dsConvertGenX0;
+        break;
+#ifndef _FULL_OPEN_SOURCE
     case VDENC_STREAMIN_HEVC:
         currKrnHeader = &kernelHeaderTable->hmeHevcVdenc;
```

Comment:
Why you don't expose HME kernel for HEVC?

Reply:
Current plan only enable for AVC VDEnc, for HEVC VDEnc not committed